packages/verticaltabs/src/react/item.tsx

```diff
@@ -190,32 +140,52 @@ const Tier1: React.FC<Tier1Props> & { Header: typeof Tier1Header } = props => (
 )
 
 const Tier1Header = React.forwardRef<any, ItemHeaderProps>((props, ref) => {
-  const { active, collapsed, collapsible, children, icon, ...rest } = props
+  const {
+    active,
+    collapsed,
+    collapsible,
+    children,
+    icon,
+    className,
+    ...rest
+  } = props
   const hideLabels = useHideLabels()
   const Tag: React.FC = wrapperProps =>
     rest.href ? (
-      <a {...wrapperProps} ref={ref as RefFor<'a'>} />
+      <a
+        {...wrapperProps}
+        ref={ref as RefFor<'a'>}
+        className={classNames(styles.tier1Header(), className)}
+      />
     ) : rest.onClick ? (
-      <button {...wrapperProps} ref={ref as RefFor<'button'>} />
+      <button
+        {...wrapperProps}
+        ref={ref as RefFor<'button'>}
+        className={classNames(styles.tier1Header(), className)}
+      />
     ) : (
-      <span {...wrapperProps} ref={ref as RefFor<'span'>} />
+      <span
+        {...wrapperProps}
+        ref={ref as RefFor<'span'>}
+        className={classNames(styles.tier1Header(), className)}
+      />
     )
   return (
-    <Tag {...styles.tier1Header()} {...rest} aria-expanded={!collapsed}>
+    <Tag {...rest} aria-expanded={!collapsed}>
       {icon &&
         React.cloneElement(icon, {
           size: CaretDownIcon.sizes.medium,
-          ...styles.itemIcon(),
+          className: classNames(styles.itemIcon()),
```

Comment:
Do we want to be able to pass a className to the icon? If so, that would need merged here.